lib/rust/ensogl/core/src/display/object/instance.rs

```diff
@@ -3012,6 +3012,14 @@ pub trait LayoutOps: Object {
         self.display_object().layout.margin.set(Vector2(margin, margin));
     }
 
+    /// Set vertical and horizontal margins of the object. Margin is the free space around the
+    /// object.
+    fn set_margin_vh(&self, vertical: impl Into<Unit>, horizontal: impl Into<Unit>) -> &Self {
+        let margin = Vector2(horizontal.into().into(), vertical.into().into());
+        self.display_object().layout.margin.set(margin);
+        self
+    }
```

Comment:
why this function is needed? This object implements `display::Object` so you can directly do `.margin.set`? 
EDIT: oh, it sets it not on root but on layout - hmm, why? There should be a proper explanation what it does and why it exists if the whole object is a display object.

Reply:
It's simply another convenience method to set margins using two values, one for verticals and one horizontals. We already have one to set all margins to given value, or to set all four independently. The vertical/horizontal one was missing from usual set you have in CSS.

Reply:
Please ignore my previous comment. I totally missed that this is addition to display objects. I was 100% sure this is in the list editor code!